src/java.base/share/classes/java/io/Reader.java

```diff
@@ -140,6 +140,124 @@ public void close() {
         };
     }
 
+    /**
+     * Returns a new {@code Reader} whose source is a {@link CharSequence}.
+     *
+     * <p> The resulting reader is not safe for use by multiple
+     * concurrent threads. If the reader is to be used by more than one
+     * thread it should be controlled by appropriate synchronization.
+     *
+     * <p> The returned reader is initially open. The reader is closed by
+     * calling the {@code close()} method. Subsequent calls to {@code close()}
+     * have no effect.
+     *
+     * <p> After the reader has been closed, the {@code read()},
```

Comment:
This paragraph is okay but the Reader methods shouldn't specify this (doing so after 25+ years of being unspecified would require being confident wouldn't invalidate any existing Readers).